setup.cfg

```diff
@@ -1,8 +1,5 @@
 [aliases]
 release = egg_info -RDb ''
-# Make sure the sphinx docs are built each time we do a dist.
-bdist = build_sphinx bdist
-sdist = build_sphinx sdist
 # Make sure the docs are uploaded when we do an upload
 upload = upload upload_docs --upload-dir build/sphinx/html
```

Comment:
This is unrelated to this PR, but we should probably remove this line, now that we are using rtd